### PR TITLE
Feature/user avatar

### PR DESCRIPTION
**Pull request by delasous**
Implements #512

### CODE REVIEW

File: packages/ui/src/UserAvatar/UserAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      </div>
+      <div className={avatarClassNames}>
+        {avatarUrl ? (
+          <img
+            className="rounded-full object-cover"
+            src={avatarUrl}
+            onError={({ currentTarget }) => {
+              currentTarget.onerror = null;
+              setAvatarUrl(undefined);
+            }}
```
Comment (ikusteu) [inline packages/ui/src/UserAvatar/UserAvatar.tsx:40]: Intersting...could you provide the resource for this (I was completely unaware of this)

**Review by ikusteu — COMMENTED** [review]
LGTM, could you just provide some resource on the `onError` thing

**Review by ikusteu — APPROVED** [review]
LGTM, could you just provide some resource on the `onError` thing

File: packages/ui/src/UserAvatar/UserAvatar.tsx (hunk 1 above)
Comment (delasous) [reply packages/ui/src/UserAvatar/UserAvatar.tsx:40]: Same here.

I found this [SO: Replace image source on error](https://stackoverflow.com/questions/34097560/react-js-replace-img-src-onerror), and the [MDN Docs: error event](https://developer.mozilla.org/en-US/docs/Web/API/Element/error_event). It is apparently a global element attribute.

**Review by ikusteu — CHANGES_REQUESTED** [review]
The component should be exported from `packages/ui/src/index.ts` so that it gets built along with the `ui` package

**Review by ikusteu — APPROVED** [review]

**Pull request merged**